### ISSUE TITLE
need serious help..............................

### ISSUE THREAD

**Issue by Gecko100** [iotaledger/wallet#744]
Cant the IOTA team post some simple youtube videos to help everyone... 

What do you do if the old seed doesn't open your wallet?  anyone give me a pointer, I will send you an IOTA or two when I get in....  

I put Iota into my light wallet 4 months ago (on my desktop) and went back yesterday to find out how they doing and discovered everything has changed (hacks, new wallets, changed seeds, recovery etc. etc.) and I have no idea what to do...

**Comment by phenomax** [comment]
Step 1: Generate a new secure seed (e.g. through https://iotaseed.io/), login through that new seed and attack your address to the tangle (Click Receiving and then hit the button)

Step 2: Open the Reclaim Tool in your IOTA Wallet

![grafik](https://user-images.githubusercontent.com/12475496/33901279-97276664-df71-11e7-85f6-5bdefe37ad50.png)

Step 3: Fill in your old seed & enter your newly generated seed into the Reclaim Tool

![grafik](https://user-images.githubusercontent.com/12475496/33901362-dee226ce-df71-11e7-8e4b-2bedc8900600.png)

**Comment by Gecko100** [comment]
Hi thanks. For step 1, do I need to login to my old wallet on my old computer? or I can login to a new wallet on any computer? I am trying to do the latter.

When I generate a new seed using the link you provided and try to login, the button flashes red with 'Connection Refused' what does this mean?

Thanks!

**Comment by chus3r** [comment]
Not a bug, recommend closing. 

Good reply @phenomax

**Comment by phenomax** [comment]
@Gecko100 No, just use any computer because your wallet is not device specific. The seed is what provides access to your wallet.
If you encounter any errors, plese try switching the public node. Usually the generated seed should work.

**Comment by Gecko100** [comment]
Many Thanks Phenomax, one last question before I close. If I tried to use the recovery tool with my old seed and using a new seed that I hadn't logged in and attached an address - is this going to be a problem? or it wouldn't have done anything? because I tried to do this initially before your instructions. Many thanks. please leave me an address for LTC or something so I can drop you a tip!
